### PR TITLE
feat: jumper scan

### PR DESCRIPTION
**Pull request by Addminus**

### CODE REVIEW

File: src/components/Menus/WalletMenu/WalletCard.tsx [hunk 1]
```diff
@@ -74,6 +78,19 @@ export const WalletCard = ({ account }: WalletCardProps) => {
...
+    account.chainId && closeAllMenus();
+    const url = `${JUMPER_SCAN_PATH}/wallet/${account.address}`;
+
+    trackEvent({
+      category: TrackingCategory.WalletMenu,
+      action: TrackingAction.OpenJumperScan,
+      label: 'open-jumper-scan-wallet',
+      disableTrackingTool: [EventTrackingTool.ARCx, EventTrackingTool.Cookie3],
+    });
+    window.open(url, '_self');
```
Comment (dennyscode) [inline src/components/Menus/WalletMenu/WalletCard.tsx:91]: @tcheee @oktapodia This `window.open` allows to navigate to the scan page with wallet-view. I had issues with `router.push`. Wdyt about this workaround?

Comment (oktapodia) [reply src/components/Menus/WalletMenu/WalletCard.tsx:91]: What issues did you get? window.open won't do anything in most of the borwser/users as it is considered intrusive for users

File: src/components/Menus/MainMenu/useMainMenuContent.tsx [hunk 2]
```diff
@@ -19,41 +17,37 @@ import { useUserTracking } from '@/hooks/userTracking/useUserTracking';
...
   const configTheme = useSettingsStore((state) => state.configTheme);
-  const { resolvedTheme, forcedTheme } = useNextTheme();
-  const { isSuperfest } = useSuperfest();
+  const { forcedTheme } = useNextTheme();
   const { isMainPaths } = useMainPaths();
   const { setSupportModalState, setSubMenuState, closeAllMenus } = useMenuStore(
     (state) => state,
   );
+  const pathname = usePathname();
+  const disabledThemeModeSwitch = pathname?.includes('/scan');
```
Comment (oktapodia) [inline src/components/Menus/MainMenu/useMainMenuContent.tsx:49]: against this, please do something similar than the superfest or profile page (as an example)

File: src/components/Menus/MainMenu/useMainMenuContent.tsx [hunk 3]
```diff
@@ -262,28 +256,21 @@ export const useMainMenuContent = () => {
       },
     },
     {
-      label: t('navbar.navbarMenu.lifiExplorer'),
+      label: 'Jumper Scan',
       prefixIcon: <SearchOutlinedIcon />,
       showMoreIcon: false,
-      link: { url: explorerUrl, external: true },
+      link: { url: '/scan', external: false },
```
Comment (oktapodia) [inline src/components/Menus/MainMenu/useMainMenuContent.tsx:262]: do not forget the trailling slash

File: src/components/Menus/WalletMenu/WalletCard.tsx [hunk 4]
```diff
@@ -74,6 +78,19 @@ export const WalletCard = ({ account }: WalletCardProps) => {
     }
   };
 
+  const handleScanButton = () => {
+    account.chainId && closeAllMenus();
+    const url = `${JUMPER_SCAN_PATH}/wallet/${account.address}`;
```
Comment (oktapodia) [inline src/components/Menus/WalletMenu/WalletCard.tsx:83]: do not forget the tralling slash

File: src/components/Navbar/Navbar.tsx [hunk 5]
```diff
@@ -29,16 +34,23 @@ export const Navbar = ({ disableNavbar = false }) => {
     if (pathname === '/gas/') {
       return;
     }
-
-    isLearnPage ? router.push(JUMPER_LEARN_PATH) : router.push('/');
+    if (isLearnPage) {
```
Comment (oktapodia) [inline src/components/Navbar/Navbar.tsx:37]: are we sure about this? that means than people cannot really go back to main page? :(

File: src/components/Navbar/NavbarButtons/WalletManagementButtons.tsx [hunk 6]
```diff
@@ -35,13 +40,16 @@ export const WalletManagementButtons = () => {
   return (
     <>
       <Box ref={walletManagementButtonsRef}>
-        {redirectToApp ? (
+        {redirectToApp && (
```
Comment (oktapodia) [inline src/components/Navbar/NavbarButtons/WalletManagementButtons.tsx:43]: Can you do this at an upper level to not load the component if not needed?

**Review by oktapodia — CHANGES_REQUESTED** [review]
couple of comments

File: src/app/[lng]/scan/[[...segments]]/page.tsx [hunk 7]
```diff
@@ -0,0 +1,57 @@
+'use client';
+import { LiFiExplorer } from '@lifi/explorer';
+import type { PaletteMode } from '@mui/material';
+import { alpha, Box, useTheme } from '@mui/material';
+import { useMemo } from 'react';
+import { ClientOnly } from 'src/components/ClientOnly';
+import { fallbackLng } from 'src/i18n';
+
+// TODO: add new button in the wallet card in wallet menu that navigates to /scan/wallet/<address>
```
Comment (tcheee) [inline src/app/[lng]/scan/[[...segments]]/page.tsx:9]: to remove when last version

File: src/components/Menus/MainMenu/useMainMenuContent.tsx (hunk 2 above)
Comment (tcheee) [reply src/components/Menus/MainMenu/useMainMenuContent.tsx:49]: agree

File: src/components/Navbar/Navbar.tsx (hunk 5 above)
Comment (tcheee) [reply src/components/Navbar/Navbar.tsx:37]: There is a big button for this. Like on Jumper Learn.

**Review by oktapodia — APPROVED** [review]
LGTM

**Pull request merged**